{"name": "feedgen", "cve_id": "CVE-2020-5227", "publish_date": "2020-01-28T23:15Z", "cwe_id": "CWE-776", "owner": "Lars Kiesow", "affected_versions": [{"version_value": "0.8.0", "operator": "<="}], "references": ["https://docs.microsoft.com/en-us/archive/msdn-magazine/2009/november/xml-denial-of-service-attacks-and-defenses", "https://github.com/lkiesow/python-feedgen/commit/f57a01b20fa4aaaeccfa417f28e66b4084b9d0cf", "https://github.com/lkiesow/python-feedgen/security/advisories/GHSA-g8q7-xv52-hf9f", "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/T6I5ENUYGFNMIH6ZQ62FZ6VU2WD3SIOI/"], "description": "Feedgen (python feedgen) before 0.9.0 is susceptible to XML Denial of Service attacks. The *feedgen* library allows supplying XML as content for some of the available fields. This XML will be parsed and integrated into the existing XML tree. During this process, feedgen is vulnerable to XML Denial of Service Attacks (e.g. XML Bomb). This becomes a concern in particular if feedgen is used to include content from untrused sources and if XML (including XHTML) is directly included instead of providing plain tex content only. This problem has been fixed in feedgen 0.9.0 which disallows XML entity expansion and external resources.", "latest_version": "0.9.0", "language": "python", "license": "BSD-2-Clause", "repository_summary": "Feed Generator (ATOM, RSS, Podcasts)", "repo_url": "https://github.com/lkiesow/python-feedgen", "impact": {"cvss2_score": 5.0, "cvss2_severity": "MEDIUM", "vector_string": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H", "av": "NETWORK", "ac": "LOW", "pr": "NONE", "c": "NONE", "i": "NONE", "a": "HIGH", "impact_score": 3.6, "exploitability_score": 3.9, "cvss3_score": 7.5, "cvss3_severity": "HIGH", "ui": "NONE", "s": "UNCHANGED"}, "ochrona_severity_score": 7.5}